### PR TITLE
Feature/edu 88 adaptible styles

### PR DESCRIPTION
**Pull request by laurentiu-ilici**

### CODE REVIEW

**Comment by ahelmberger** [comment]
Nice :+1:  (I just tried it in green)

**Review by ahelmberger — APPROVED** [review]

File: test-app/main.less [hunk 1]
```diff
@@ -1 +1,21 @@
 @import url('../dist/styles/main.less');
+
+//Example font family override
+@global-font-family: 'Roboto', sans-serif;
+
+//Example header size overrides
+//@global-h1-font-size: 5em;
+//@global-h2-font-size: 2.5em;
```
Comment (adrianaluchian) [inline test-app/main.less:8]: Let's remove commented out code please 🙏

Comment (ahelmberger) [reply test-app/main.less:8]: Or make this THE code, so we have some more obvious overrides in the test app?

Comment (laurentiu-ilici) [reply test-app/main.less:8]: I left this here as a way to document and an easy way for us to try out. The problem with uncommenting this all the time is that we lose how elmu really looks  like. I'll make it the code.

**Review by ahelmberger — APPROVED** [review]

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**